docker/docker-compose-with-cassandra.yml

```diff
@@ -24,7 +24,7 @@ services:
   datahub-actions:
     container_name: datahub-actions
     hostname: actions
-    image: acryldata/datahub-actions:${ACTIONS_VERSION:-head}
+    image: acryldata/datahub-actions${ACTIONS_BASE:}:${ACTIONS_VERSION:-head}
```

Comment:
let's just make the whole thing configuration e.g. `DATAHUB_ACTIONS_IMAGE`, and have it default to `acryldata/datahub-actions`

That's similar to our approach for all other images like `DATAHUB_FRONTEND_IMAGE`